robot/rover/MobilePlatform/MobilePlatform.ino

```diff
@@ -19,6 +21,12 @@
 #define FRONT_BASE_DEFAULT_PWM 65
 #define REAR_BASE_DEFAULT_PWM 35
 
+/* Activity Indicator */
+#define PIN 15
```

Comment:
please use a more descriptive variable name, especially when it's a global macro that could end up anywhere

Reply:
also, please put pin definitions (you have two) in pins.h